collector/rpc_metrics.go

```diff
@@ -222,3 +224,22 @@ func (collector *NodeRpcMetrics) Collect(ch chan<- prometheus.Metric) {
 		}
 	}
 }
+
+func (collector *NodeRpcMetrics) RecordValidators() {
+	for {
+		fmt.Println("do the thing")
+		srExt, err := collector.externalClient.Get("status", nil)
+		if err != nil {
+			fmt.Println("Failed to get status from external endpoint.")
```

Comment:
Can we log the error here as well?
```suggestion
			fmt.Printf("Failed to get status from external endpoint: %s\n", err.Error())
```

Reply:
Same for down below. This will only tell us where the code is breaking, not how it's breaking (which this pattern has caused me issues before with the ontology and eos exporters at least)